tools/confix/cmd/migrate.go

```diff
@@ -21,31 +22,42 @@ var (
 
 func MigrateCommand() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "migrate [target-version] <app-toml-path> (options)",
-		Short: "Migrate Cosmos SDK app configuration file to the specified version",
-		Long: `Migrate the contents of the Cosmos SDK app configuration (app.toml) to the specified version.
+		Use:   "migrate [target-version] <config-path> [config-type]",
+		Short: "Migrate Cosmos SDK configuration file to the specified version",
+		Long: `Migrate the contents of the Cosmos SDK configuration (app.toml or client.toml) to the specified version. Configuration type is app by default.
 The output is written in-place unless --stdout is provided.
 In case of any error in updating the file, no output is written.`,
 		Args: cobra.MinimumNArgs(1),
 		RunE: func(cmd *cobra.Command, args []string) error {
-			var filename string
+			var configPath string
+			
 			clientCtx := client.GetClientContextFromCmd(cmd)
-			switch {
+			targetVersion := args[0]
+			configType := confix.AppConfigType // Default to app configuration
+
+			if len(args) > 2 {
+				configType = strings.ToLower(args[2])
+				if configType != confix.AppConfigType && configType != confix.ClientConfigType {
```

Comment:
Please hoist this validation out of this if statement and put it down after it so that there will be unconditional validation of the configType.